setup.py

```diff
@@ -0,0 +1,24 @@
+
+from setuptools import setup, find_packages
+
+setup(
+    name='agentless-system-crawler',
+    version='0.0.1.dev',
+    #description='',
```

Comment:
I don't know how to fill this out. There isn't really anything in the README either...